tests/flow.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::iter_cloned_collect)]
```

Comment:
You can replace all `.iter().copied()` by `into_iter` and remove all `#![allow(clippy::iter_cloned_collect)]`

Reply:
This doesn't work for all of them, since sometimes to underlying values do need to be cloned, but I've fixed the ones for where it doesn't need to be